sql-join/what-and-when.sql

```diff
@@ -0,0 +1,7 @@
+SELECT "films"."releaseYear" AS "releaseYear",
+       "films"."title"       AS "title",
+       "categories"."name"   AS "name"
+  FROM "films"
+  JOIN "filmCategory" USING ("filmId")
+  JOIN "categories" USING ("categoryId")
+ WHERE "title" = 'Boogie Amelie';
```

Comment:
In your `WHERE` clauses, it's generally good to specify which table the column is in.